Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -275,6 +275,9 @@ public struct __CommandLineArguments_v0: Sendable {
 
   /// The value of the `--repeat-until` argument.
   public var repeatUntil: String?
+
+  /// The value of the `--experimental-attachment-path` argument.
+  public var experimentalAttachmentPath: String?
```

Comment:
An analogous property on `Configuration` is named "attachment**Directory**Path". Two questions:

- Should it include the word "directory" to help indicate it's intended to be a directory, not the path to a single attachment file (as well as for consistency with the `Configuration` property)?
- Should we consider making the word attachment plural, to help emphasize this directory will be the place where potentially multiple attachments will be placed?

These comments also apply to the CLI flag this property corresponds to

Reply:
I don't feel too strongly about it. If you do, let me know what name you'd prefer here and I'll just use it.

Reply:
Discussed off-GitHub.